### ISSUE TITLE
check-postgres-connections -w AND -c

### ISSUE THREAD

**Issue by mnietz** [sensu-plugins/sensu-plugins-postgres#17]
When i try to specify -w and -c together the plugin reports a warning state when it should report a critical one.

Example:

```
check-postgres-connections.rb -h localhost -d template1 -u sensu -p sensu -w 5 -c 6
CheckPostgresConnections WARNING: PostgreSQL connections at 8 out of 10 connections
```

Specifying only -w or -c works as expected. It doesn't make any difference using -a or not. 

Sensu: 0.25.7 latest version of the plugin.

**Comment by alex-pw** [comment]
Thanks for the great plugins!  I can confirm this behavior.  Critical value needs to be checked prior to warning value.  Workaround is to use only -c as @mnietz reports.

**Comment by nevins-b** [comment]
This is addressed in #24

**Comment by majormoses** [comment]
closed per https://github.com/sensu-plugins/sensu-plugins-postgres/pull/24
